WMF Framework/Configuration.swift

```diff
@@ -95,6 +95,13 @@ public class Configuration: NSObject {
         static let wikimedia = "wikimedia.org"
         static let metaWiki = "meta.wikimedia.org"
         static let wikimediafoundation = "wikimediafoundation.org"
+        static let uploads = "upload.wikimedia.org"
+    }
+    
+    public struct File {
+        public struct Extensions {
+            public static let oggAudio = Set<String>(["ogg", "oga"])
```

Comment:
Is the intent behind making this a `Set` instead of an `Array` just for optimization sake? While I can see how `Set`'s technically the _more_ correct of the two data structures to use here, I wonder if there's value in it just being a plain ole `Array` to avoid unintended confusion as to why it requires the characteristics of a `Set` and to remove cruft from the definition. Since the data is well-bounded, accidentally having "ogg" or "oga" listed more than once isn't going to affect the time complexity or functionality of `contains(...)` in any meaningful way. Plus, we don't get a compiler warning or anything for accidentally defining duplicate `Set` items.

Feel free to ignore, was mostly just curious if I was missing something about the intent.

Reply:
I made it a set for optimization and to signal that it's meant for lookup. It's entirely possible the set is slower than the array with only two elements. I'm fine with changing it to an array.

Reply:
I wasn't really happy with having it in that struct anyway, and it's unlikely it'll be used elsewhere. I moved it into a private extension closer to where it's needed and switched it to an equality check.